src/modules/tasks.js

```diff
@@ -0,0 +1,38 @@
+export const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+function storeTasks() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+export function checkTasks() {
+  const lists = document.querySelector('.lists');
+
+  tasks.sort((a, b) => a.index - b.index);
+
+  tasks.forEach((task) => {
+    lists.insertAdjacentHTML('beforeend', `<li class="task-item" contentEditable = "false"><input type="checkbox">${task.description}<i id="task-btn" class="fa-solid fa-ellipsis-vertical"></i></li>`);
+  });
+}
+
+export function addTask(description) {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length,
+  };
+  tasks.push(task);
+  storeTasks();
```

Comment:
- [ ] Great job here, I was able to add tasks to the local storage 👍, but let's make some changes so that we meet some requirements for this project when a task is added, the index should be 1 and not zero, so if I have five items, the index of the fifth item should be 5, not 4.

![add](https://github.com/RicardoEgas/To-Do-List/assets/53124548/b72046e3-cf68-4263-ae14-ee79d34be54b)